openhcl/virt_mshv_vtl/src/processor/hardware_cvm/mod.rs

```diff
@@ -431,12 +426,16 @@ impl<T, B: HardwareIsolatedBacking> hv1_hypercall::VtlReturn for UhHypercallHand
 
         self.vp.unlock_tlb_lock(Vtl::Vtl1);
 
+        let hv = &mut self.vp.backing.cvm_state_mut().hv[GuestVtl::Vtl1];
+        if hv.synic.vina().auto_reset() {
+            hv.set_vina_asserted(false).unwrap();
```

Comment:
Stephanie and I are still thinking through the proper error handling for here and vtl_call, this will be addressed.